### ISSUE TITLE
Completion support in 'from . import'

### ISSUE THREAD

**Issue by hanpari** [Microsoft/python-language-server#594]
I am unable to run relative import in my vscode with PLS 
When I write

    from . import 

intellisense had no idea what to offer.

I have the same trouble on both of my working stations with win10. It is not new issue. I was never able to get it correctly every time I have tried.

Edit
Corrected syntax

Correct relative import with jedi
![jedi_relative_import](https://user-images.githubusercontent.com/7354817/52637374-5de66900-2ecf-11e9-8176-b318e7675eca.png)

Incorrect with PLS
![pls_relative_import](https://user-images.githubusercontent.com/7354817/52637387-65a60d80-2ecf-11e9-94c0-d6b77809a734.png)

**Comment by jakebailey** [comment]
That isn't valid Python import syntax. Imports look like:

```
import foo
from .bar import Bar
import something as sm
from lib import xyz as zyx, abc as cba
from hello import *
```

There is no `import _ from` style import. Try writing it another way and see if you get completion.

https://docs.python.org/3/reference/simple_stmts.html?highlight=import#grammar-token-import-stmt

**Comment by hanpari** [comment]
Sorry my bad I made mistake. But it is too early to close the issue. Of course I am using valid syntax and jedi has no problem to offer intellisense.

**Comment by MikhailArkhipov** [comment]
Let me look at completion code first

**Comment by hanpari** [comment]
OK, I have switched to older release of VS Code and now PLS works correctly.
Now I am using

> Version: 1.29.1
> Commit: bc24f98b5f70467bc689abf41cc5550ca637088e
> Date: 2018-11-15T19:13:36.375Z
> Electron: 2.0.12
> Chrome: 61.0.3163.100
> Node.js: 8.9.3
> V8: 6.1.534.41
> Architecture: x64

**Comment by hanpari** [comment]
I have tried fresh installation after complete C format. The buggy behaviour  remains the same. Even Jedi doesnt work anymore.

**Comment by MikhailArkhipov** [comment]
`requests` was fixed in https://github.com/Microsoft/python-language-server/issues/462, will be in the 0.2 of LS. The second is support for completion in `from . import `.

**Comment by jakebailey** [comment]
In the new LS, both `from ` and `from .` cause the path resolver to output `ImplicitPackageImport` with `FullName == null` and `Name = "C:\\Python37\\python37.zip"`. This fails as `FullName` is used as a key in the ImportHandler to look up a module. 

This causes an exception in two places:

- During analysis, the exception is thrown in both cases but is eaten and never reported (???)
- For the completion, the analysis runs again, but in this case the exception is not eaten and instead is exposed as an RPC exception.

```
[Error - 1:38:57 PM] Request textDocument/completion failed.
  Message: Value cannot be null.
Parameter name: key
  Code: -32000 
   at System.Collections.Generic.Dictionary`2.FindEntry(TKey key)
   at System.Collections.Generic.Dictionary`2.TryGetValue(TKey key, TValue& value)
   at Microsoft.Python.Analysis.Analyzer.Handlers.ImportHandler.GetOrCreateVariableModule(String& fullName, PythonVariableModule& parentModule, String& memberName) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\Handlers\ImportHandler.cs:line 167
   at Microsoft.Python.Analysis.Analyzer.Handlers.ImportHandler.TryGetPackageFromImport(ImplicitPackageImport& implicitPackageImport, PythonVariableModule& parentModule, PythonVariableModule& variableModule) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\Handlers\ImportHandler.cs:line 155
   at Microsoft.Python.Analysis.Analyzer.Handlers.ImportHandler.HandleImportSearchResult(IImportSearchResult& imports, PythonVariableModule& parent, NameExpression& asNameExpression, LocationInfo& location, PythonVariableModule& variableModule) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\Handlers\ImportHandler.cs:line 88
   at Microsoft.Python.Analysis.Analyzer.Handlers.ImportHandler.HandleFromImport(FromImportStatement node) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\Handlers\FromImportHandler.cs:line 42
   at Microsoft.Python.Analysis.Analyzer.AnalysisWalker.Walk(FromImportStatement node) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\AnalysisWalker.cs:line 74
   at Microsoft.Python.Parsing.Ast.FromImportStatement.Walk(PythonWalker walker) in C:\Users\jabaile\python-language-server\src\Parsing\Impl\Ast\FromImportStatement.cs:line 58
   at Microsoft.Python.Parsing.Ast.SuiteStatement.Walk(PythonWalker walker) in C:\Users\jabaile\python-language-server\src\Parsing\Impl\Ast\SuiteStatement.cs:line 38
   at Microsoft.Python.Parsing.Ast.PythonAst.Walk(PythonWalker walker) in C:\Users\jabaile\python-language-server\src\Parsing\Impl\Ast\PythonAst.cs:line 80
   at Microsoft.Python.Analysis.Analyzer.PythonAnalyzer.Analyze(IDependencyChainNode`1 node, Int32 version, Stopwatch stopWatch, CancellationToken cancellationToken) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\PythonAnalyzer.cs:line 274
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Python.Analysis.Analyzer.PythonAnalyzerEntry.<>c.<GetAnalysisAsync>b__27_0(Task`1 t) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\PythonAnalyzerEntry.cs:line 48
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.Python.Analysis.Analyzer.PythonAnalyzer.GetAnalysisAsync(PythonAnalyzerEntry entry, CancellationToken timeoutCt, CancellationToken cancellationToken) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\PythonAnalyzer.cs:line 91
   at Microsoft.Python.Analysis.Analyzer.PythonAnalyzer.GetAnalysisAsync(IPythonModule module, Int32 waitTime, CancellationToken cancellationToken) in C:\Users\jabaile\python-language-server\src\Analysis\Ast\Impl\Analyzer\PythonAnalyzer.cs:line 84
   at Microsoft.Python.LanguageServer.Implementation.Server.Completion(CompletionParams params, CancellationToken cancellationToken) in C:\Users\jabaile\python-language-server\src\LanguageServer\Impl\Implementation\Server.Editor.cs:line 36
   at Microsoft.Python.LanguageServer.Implementation.LanguageServer.Completion(JToken token, CancellationToken cancellationToken) in C:\Users\jabaile\python-language-server\src\LanguageServer\Impl\LanguageServer.cs:line 239
```

**Comment by AlexanderSher** [comment]
Fixed in #722
